content/docs/dvclive/live/monitor_system.md

```diff
@@ -0,0 +1,159 @@
+# Live.monitor_system()
+
+DVCLive can track and log your system metrics so you can see your hardware
+performance and limitations during your experiments. The metrics concerns your
+CPU, RAM and disk memory usage. It can also monitor your GPUs usage, if you have
+some.
+
+```py
+def monitor_system(
+    interval: float = 0.05,
+    num_samples: int = 20,
+    directories_to_monitor: Optional[Dict[str, str]] = None,
```

Comment:
See https://github.com/iterative/dvclive/pull/785/files#r1499473837. Should it be `{"main": "/"}`?